README.rst

```diff
@@ -20,72 +20,114 @@ Overview
 More information about KQueen Architecture and use cases is described in `RATIONALE <RATIONALE.md>`_ file.
 
 Requirements
------------
+------------
 
 -  Python v3.6 and higher.
 -  Pip v3 and higher.
 -  Docker stable release (v17.03 and higher is preferable).
 -  Docker-compose stable release (v1.16.0 and higher is preferable).
-
-::
-
-    mkvirtualenv -p /usr/bin/python3 kqueen
+-  And some linux packages: libsasl2-dev python-dev libldap2-dev libssl-dev
```

Comment:
These packages are working only for Debian/Ubuntu systems. Only `openldap-devel` was required on my Fedora.

Reply:
Required packages are usually not listed in documentation for this very reason, they depend on the distribution you use, while the rest of the application works on all of them. So devs setting up their envs usually need to figure them out for themselves. You can mark these packages as example for Ubuntu 16.04, but they will differ from distro to distro.